src/components/step/ConnectDeviceAccountStepContainer/ConnectDeviceAccountStepContainer.tsx

```diff
@@ -2,13 +2,26 @@ import React, { useState, useEffect } from "react";
 import MyDataHelps, { StepConfiguration } from "@careevolution/mydatahelps-js";
 import ConnectDeviceAccountStep from "../ConnectDeviceAccountStep";
 
-export default function () {
+export interface ConnectDeviceAccountStepContainerProps {
+    deviceType?: string;
+    providerID?: number;
+}
+
+export default function (props: ConnectDeviceAccountStepContainerProps) {
     const [title, setTitle] = useState<string>();
     const [text, setText] = useState<string>();
     const [styles, setStyles] = useState<any>({});
-    const [deviceType, setDeviceType] = useState<string>("");
-    const [providerID, setProviderID] = useState<number>();
-    const [connected, setConnected] = useState<boolean>();
+    const [deviceType, setDeviceType] = useState<string>(props.deviceType ?? "");
+    const [providerID, setProviderID] = useState<number>(props.providerID ?? 0);
+
+    function onConnect() {
```

Comment:
Why is this moved to the container?  

I think the idea with the container is that it should just be thin (no stories) and just get the config / styles and render the step...ultimately this pattern is a bit wonky though and I am regretting it in favor of having some sort of "previewState" or "designerMode" flag like the others.  But...maybe neither here nor there for now

Reply:
I was under the impression that the containers are for behavior, whereas the UI components are just for presentation.